### DISCUSSION TITLE
Python versions

### DISCUSSION THREAD

**Discussion by M0rtenB** [astral-sh/rye#1223] [General]
Will Python 3.12.4 eventually show up? 5 weeks old now.
And will the new free-threaded version be available as a toolchain?

**Comment by philippemiron** [comment]
There is an issue tracking this: https://github.com/indygreg/python-build-standalone/issues/273.

**Comment by S64** [comment]
I checked today and the latest rye has `cpython@3.12.4` available!

```sh
rye --version
# rye 0.37.0
# commit: 0.37.0 (09b67c469 2024-07-20)
# platform: macos (aarch64)
# self-python: cpython@3.12.3
# symlink support: true
# uv enabled: true
rye toolchain list --include-downloadable | grep cpython@3.12.4
# cpython@3.12.4 (downloadable)
```

**Comment by zanieb** [comment]
Ah yeah we published this recently. Thanks for the reminder!
